src/operations.rs

```diff
@@ -156,6 +157,7 @@ pub(crate) fn aesdec(value: u128, xor: u128) -> u128 {
     use core::arch::aarch64::*;
     #[cfg(target_arch = "arm")]
     use core::arch::arm::*;
+    use core::mem::transmute;
     unsafe {
         let value = transmute!(value);
         xor ^ transmute::<_, u128>(vaesimcq_u8(vaesdq_u8(value, transmute!(0u128))))
```

Comment:
I'd recommend using `transmute!` so we can remove the `unsafe` here:

```suggestion
    let value = transmute!(value);
    let value: u128 = transmute!(vaesimcq_u8(vaesdq_u8(value, transmute!(0u128))))
    xor ^ value
```